common/scripted_effects/cwp_political_setup.txt

```diff
@@ -77,8 +71,13 @@ setup_starting_ideologies = {
 			limit = {
 				owner = {
 					or = {
-						has_law = law_type:law_no_nationalization
+						has_law = law_type:law_socialization
+					}
+					and = {
 						has_law = law_type:law_public_utility_companies
+						has_law = law_type:law_no_nationalization
+						has_law = law_type:law_resource_extraction_nationalization
+						has_law = law_type:law_military_industry_nationalization
```

Comment:
similar to a previous comment. it's impossible to have all these laws at once. did you mean `or`?